providers/certification-client/units/client-cert-desktop-24-04.pxu

```diff
@@ -35,7 +35,7 @@ nested_part:
     led-cert-manual
     mediacard-cert-manual
     memory-manual
-    mobilebroadband-cert-manual
+    wwan-manual
```

Comment:
it may need to add the `after-suspend-wwan-manual`, although it didn't test the after suspend in mobilebroadband.

Reply:
I saw some manual  cert test plan didn't include the `after-suspend-wwan-manual`.
`providers/certification-client/units/client-cert-desktop-22-04.pxu`
`providers/certification-client/units/client-cert-odm-desktop-20-04.pxu`
`providers/certification-client/units/client-cert-odm-desktop-22-04.pxu`

Reply:
the jobs in after-suspend-wwan-manual are:

```
    after-suspend-wwan/detect-manual
    after-suspend-wwan/check-sim-present-manual
    after-suspend-wwan/gsm-connection-interrupted-manual
```

All of these have been superseded by automated jobs, I think that's why they are not present.

Reply:
Agree!! We could also remove `wwan-manual` in cert test plan.
```
id: wwan-manual
unit: test plan
_name: Manual wwan tests
_description: Manual wwan tests for Snappy Ubuntu Core devices
include:
    wwan/detect-manual
    wwan/check-sim-present-manual
    wwan/gsm-connection-interrupted-manual
```

Reply:
Or keep it (and re-introduce after-suspend-wwan-manual as well), but remove the superseded jobs in the test plan definitions...

Reply:
In the end I introduced `after-suspend-wwan-manual` where it made sense, but I removed the manual jobs that have been superseded by automated jobs... which means all of them :D However, it's still useful to keep these nested parts if we ever write another manual job for WWAN testing.